src/Tools/dotnet-trace/CommandLine/Commands/ConvertCommand.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Tools.RuntimeClient;
+using System;
+using System.IO;
+using System.CommandLine;
+using System.CommandLine.Builder;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Tools.Trace
+{
+    internal static class ConvertCommandHandler
+    {
+        public static int ConvertFile(IConsole console, FileInfo inputFilename, TraceFileFormat format, FileInfo output)
+        {
+            try
+            {
+                if (format == TraceFileFormat.netperf)
+                    throw new ArgumentException("Cannot convert to netperf format.");
+                
+                if (!inputFilename.Exists)
+                    throw new FileNotFoundException($"File '{inputFilename}' does not exist.");
```

Comment:
You can use `ArgumentExtensions.ExistingOnly` in your parser configuration and get this validation done for you upstream.

Reply:
Bit of a non sequitur and it may be something wrong with Omnisharp on mac or my config, but I don't get any intellisesnse for System.CommandLine in VSCode on Mac.  Any ideas on why that may be?

Reply:
Not a clue. Other libraries aren't affected?

@rchande Ring any bells?

Reply:
May be the build scripts for this repo fighting with Omnisharp though, since there are other things messed up.  I might ping offline if it is still an issue later.